project.godot

```diff
@@ -31,6 +31,8 @@ window/stretch/mode="canvas_items"
 [global_group]
 
 players=""
+switch=""
+animator=""
```

Comment:
I think this (and the addition of `switch.tscn`) are also not meant to be part of this PR.